docs/hardware.rst

```diff
@@ -10,17 +10,17 @@ Required Hardware
 -----------------
 
 Servers
-~~~~~~~
+^^^^^^^
 
 These are the core components of a SecureDrop instance.
 
--  *Application Server*: 1 physical server to run the SecureDrop web
-   services.
--  *Monitor Server*: 1 physical server which monitors activity on the
-   *Application Server* and sends email notifications to an
-   administrator.
--  **Network Firewall**: 1 physical computer that is used as a dedicated
-   firewall for the SecureDrop servers.
+* *Application Server*: 1 physical server to run the SecureDrop web services.
+
+* *Monitor Server*: 1 physical server which monitors activity on the
+  *Application Server* and sends email notifications to an administrator.
+
+* **Network Firewall**: 1 physical computer that is used as a dedicated firewall
```

Comment:
FWIW, we have a docs style guide: https://docs.securedrop.org/en/latest/development/documentation_guidelines.html#style-guide, which is relevant to issues like this.

Note that the docs style guide is fairly new and incomplete - there is certainly room for improvement (additions and/or modifications). **However**, any such changes should happen in separate PR's focused specifically on those changes, and this PR should use the currently documented style recommendations.